docs/guidance/tls.md

```diff
@@ -0,0 +1,140 @@
+# TLS Authentication
+
+Ray can be configured to use TLS on it’s gRPC channels. This means that 
+connecting to the Ray client on the head node will require an appropriate 
```

Comment:
Is this referring to [Ray Client](https://docs.ray.io/en/latest/cluster/running-applications/job-submission/ray-client.html), or is this talking about ordinary internal gRPC communication between nodes?  If it's the former, we can link to the Ray Client doc, but if not, maybe we should avoid the term "Ray Client"

Reply:
I just copied and pasted this paragraph from this [document](https://docs.ray.io/en/latest/ray-core/configure.html?highlight=TLS#tls-authentication).

> Ray can be configured to use TLS on it’s gRPC channels. This means that connecting to the Ray client on the head node will require an appropriate set of credentials and also that data exchanged between various processes (client, head, workers) will be encrypted.

Reply:
Updated https://github.com/ray-project/kuberay/pull/989/commits/0cfe9df3fe5fb17a2cd01180a06eada3a3adda51